src/scripts/admin/admin.store.js

```diff
@@ -90,6 +92,90 @@ let UserStore = Reflux.createStore({
     },
 
     /**
+     * Search username and email
```

Comment:
I would breaker these two filtering functions into three functions. One for handling the search input, one for handling the admin button filter click and a shared filtering function.

	searchUser (input) {
		this.filter(input, this.data.adminFilter);
	},

	filterAdmin () {
		this.filter(this.data.searchInput, !this.data.adminFilter);
	},

	filter (searchInput, adminFilter) {
		for (let user of users) {
			user.visible = true;

			// search filtering
			if (/* search string does not match user and search string not empty */) {
				user.visible = false;
			}

			// button filtering
			if (/* admin filter on and user not admin */) {
				user.visible = false;
			}


		}

		this.update({users, searchInput, adminFilter});
	}